src/github3/__init__.py

```diff
@@ -59,6 +59,7 @@
     'GitHubError',
     'GitHubStatus',
     'authorize',
+    'checks',
```

Comment:
It may not be needed. I thought I had a problem though with a relative import in a different file, but it could have been a different issue. I'll remove it and see if tests still pass 😁

Reply:
Narrator: They passed.